src/protocols/rdp/user.c

```diff
@@ -121,13 +121,16 @@ int guac_rdp_user_join_handler(guac_user* user, int argc, char** argv) {
 
         /* Inbound arbitrary named pipes */
         user->pipe_handler = guac_rdp_pipe_svc_pipe_handler;
-        
+
         /* If we own it, register handler for updating parameters during connection. */
         if (user->owner)
             user->argv_handler = guac_argv_handler;
 
     }
 
+    /* Process some resources when users leaving */
```

Comment:
What do you mean by "process some resources"?

Reply:
This indicates the tasks done by `guac_rdp_user_leave_handler()` function.